### DISCUSSION TITLE
Do you have a planning for future GE 4+ release dates?

### DISCUSSION THREAD

**Discussion by cdo29** [gitextensions/gitextensions#11645] [Q&A] [answered]
Hi,

first of all thank you for your great job.
I use GE at work and I am interested in recent improvements you made on the master branch.
Do you have a discution somewhere where future release dates are displayed.
I am not looking for an acurate date but just want to know if you plan a new release this in the coming weeks for exemple.

Cheers,
Cédric

**Comment by gerhardol** [comment, marked as answer]
There is no set plan. There are a number of PRs in progress right now, a release will more or less stop new features so they are not done too often. There are some plugin interfaces that need to be resolved too.

Some more discussions: https://github.com/gitextensions/gitextensions/discussions/11573

**Reply by cdo29** [reply]
Thank you for the information.
Have a good day

**Comment by btzdnl** [comment]
I very much want a new release, too. The broken Git history display (#10970) is super annoying, it’s been broken for almost a year now.

**Reply by RussKie** [reply]
We hear you. A release is a somewhat involved and a disruptive process. And the team right now is in the "zone" implementing new features and improving existing implementations. It'd be rude to interrupt the flow. Thank you for your eagerness and your patience.

You're welcome to trial [nightly dev builds](https://ci.appveyor.com/project/gitextensions/gitextensions/branch/master/artifacts) (which are generally quite stable), with that you will be helping us to iron out any bugs thus making the upcoming release smoother and more stable.

**Reply by cdo29** [reply]
Thank you very much :)
That is very helpful
